etc/supervisor/conf.d/web.conf

```
@@ -1,6 +1,5 @@
-[program:uwsgi]
-command = /usr/bin/uwsgi --ini /etc/uwsgi/apps-enabled/weblate.ini
-environment = UWSGI_DEB_CONFNAME="weblate",UWSGI_DEB_CONFNAMESPACE="app"
+[program:gunicorn]
+command = /usr/local/bin/gunicorn weblate.wsgi:application --workers=%(ENV_GUNICORN_WORKERS)s --access-logfile='-' --error-logfile='-' --forwarded-allow-ips="*" --bind unix:///run/gunicorn/app/weblate/socket
```

For performance reasons, it might be good to add the following:

```suggestion
command = /usr/local/bin/gunicorn weblate.wsgi:application --workers=%(ENV_GUNICORN_WORKERS)s --access-logfile='-' --error-logfile='-' --forwarded-allow-ips="*" --bind unix:///run/gunicorn/app/weblate/socket --preload --max-requests 10000 --max-requests-jitter 1000
```